packages/integration-karma/test/shadow-dom/EventTarget-methods/EventTarget.dispatchEvent.spec.js

```diff
@@ -0,0 +1,24 @@
+describe('EventTarget.dispatchEvent', () => {
+    it('should return true when "Event.preventDefault" is not invoked', () => {
+        const target = new EventTarget();
```

Comment:
This will fail on IE11, the `EventTarget` constructor was added after. You need to guard it.
Just as sanity check, I would also recommend creating a test when the EventTarget is an `Element` to verify is the behavior is consistent.